crates/bevy_pbr/src/meshlet/visibility_buffer_raster_node.rs

```diff
@@ -311,10 +314,11 @@ impl Node for MeshletVisibilityBufferRasterPassNode {
                 resolve_depth_shadow_view_pipeline,
                 camera,
             );
-            downsample_depth(
+            meshlet_view_resources.depth_pyramid.downsample_depth(
+                "main downsample depth",
```

Comment:
The labels for the meshlet shadow pass downsample seem wrong to me.

Reply:
Ah, I didn't realize that was for shadows. Makes sense. Should be fixed now